src/tests/detect-http-uri.c

```diff
@@ -6793,8 +6613,6 @@ static void DetectHttpUriRegisterTests (void)
 
     UtRegisterTest("UriTestSig28", UriTestSig28);
     UtRegisterTest("UriTestSig29", UriTestSig29);
-    UtRegisterTest("UriTestSig30", UriTestSig30);
-    UtRegisterTest("UriTestSig31", UriTestSig31);
```

Comment:
Just to double check: is this the commit that added similar logic elsewhere? https://github.com/OISF/suricata/pull/9974/commits/32b4423d454e174ace618ac874d6f4ee920407ce

Reply:
heh, no, it's in a branch that is still WIP. I'll drop this commit from here.